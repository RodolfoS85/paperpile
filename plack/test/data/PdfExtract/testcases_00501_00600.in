---
file: ../../../PDFREPOSITORY/000501.pdf
---
file: ../../../PDFREPOSITORY/000502.pdf
---
file: ../../../PDFREPOSITORY/000503.pdf
---
file: ../../../PDFREPOSITORY/000504.pdf
---
file: ../../../PDFREPOSITORY/000505.pdf
---
file: ../../../PDFREPOSITORY/000506.pdf
---
file: ../../../PDFREPOSITORY/000507.pdf
---
file: ../../../PDFREPOSITORY/000508.pdf
---
file: ../../../PDFREPOSITORY/000509.pdf
---
file: ../../../PDFREPOSITORY/000510.pdf
---
file: ../../../PDFREPOSITORY/000511.pdf
---
file: ../../../PDFREPOSITORY/000512.pdf
---
file: ../../../PDFREPOSITORY/000513.pdf
---
file: ../../../PDFREPOSITORY/000514.pdf
---
file: ../../../PDFREPOSITORY/000515.pdf
---
file: ../../../PDFREPOSITORY/000516.pdf
---
file: ../../../PDFREPOSITORY/000517.pdf
---
file: ../../../PDFREPOSITORY/000518.pdf
---
file: ../../../PDFREPOSITORY/000519.pdf
---
file: ../../../PDFREPOSITORY/000520.pdf
---
file: ../../../PDFREPOSITORY/000521.pdf
---
file: ../../../PDFREPOSITORY/000522.pdf
---
file: ../../../PDFREPOSITORY/000523.pdf
---
file: ../../../PDFREPOSITORY/000524.pdf
---
file: ../../../PDFREPOSITORY/000525.pdf
---
file: ../../../PDFREPOSITORY/000526.pdf
---
file: ../../../PDFREPOSITORY/000527.pdf
---
file: ../../../PDFREPOSITORY/000528.pdf
---
file: ../../../PDFREPOSITORY/000529.pdf
---
file: ../../../PDFREPOSITORY/000530.pdf
---
file: ../../../PDFREPOSITORY/000531.pdf
---
file: ../../../PDFREPOSITORY/000532.pdf
---
file: ../../../PDFREPOSITORY/000533.pdf
---
file: ../../../PDFREPOSITORY/000534.pdf
---
file: ../../../PDFREPOSITORY/000535.pdf
---
file: ../../../PDFREPOSITORY/000536.pdf
---
file: ../../../PDFREPOSITORY/000537.pdf
---
file: ../../../PDFREPOSITORY/000538.pdf
---
file: ../../../PDFREPOSITORY/000539.pdf
---
file: ../../../PDFREPOSITORY/000540.pdf
---
file: ../../../PDFREPOSITORY/000541.pdf
---
file: ../../../PDFREPOSITORY/000542.pdf
---
file: ../../../PDFREPOSITORY/000543.pdf
---
file: ../../../PDFREPOSITORY/000544.pdf
---
file: ../../../PDFREPOSITORY/000545.pdf
---
file: ../../../PDFREPOSITORY/000546.pdf
---
file: ../../../PDFREPOSITORY/000547.pdf
---
file: ../../../PDFREPOSITORY/000548.pdf
---
file: ../../../PDFREPOSITORY/000549.pdf
---
file: ../../../PDFREPOSITORY/000550.pdf
---
file: ../../../PDFREPOSITORY/000551.pdf
---
file: ../../../PDFREPOSITORY/000552.pdf
---
file: ../../../PDFREPOSITORY/000553.pdf
---
file: ../../../PDFREPOSITORY/000554.pdf
---
file: ../../../PDFREPOSITORY/000555.pdf
---
file: ../../../PDFREPOSITORY/000556.pdf
---
file: ../../../PDFREPOSITORY/000557.pdf
---
file: ../../../PDFREPOSITORY/000558.pdf
---
file: ../../../PDFREPOSITORY/000559.pdf
---
file: ../../../PDFREPOSITORY/000560.pdf
---
file: ../../../PDFREPOSITORY/000561.pdf
---
file: ../../../PDFREPOSITORY/000562.pdf
---
file: ../../../PDFREPOSITORY/000563.pdf
---
file: ../../../PDFREPOSITORY/000564.pdf
---
file: ../../../PDFREPOSITORY/000565.pdf
---
file: ../../../PDFREPOSITORY/000566.pdf
---
file: ../../../PDFREPOSITORY/000567.pdf
---
file: ../../../PDFREPOSITORY/000568.pdf
---
file: ../../../PDFREPOSITORY/000569.pdf
---
file: ../../../PDFREPOSITORY/000570.pdf
---
file: ../../../PDFREPOSITORY/000571.pdf
---
file: ../../../PDFREPOSITORY/000572.pdf
---
file: ../../../PDFREPOSITORY/000573.pdf
---
file: ../../../PDFREPOSITORY/000574.pdf
---
file: ../../../PDFREPOSITORY/000575.pdf
---
file: ../../../PDFREPOSITORY/000576.pdf
---
file: ../../../PDFREPOSITORY/000577.pdf
---
file: ../../../PDFREPOSITORY/000578.pdf
---
file: ../../../PDFREPOSITORY/000579.pdf
---
file: ../../../PDFREPOSITORY/000580.pdf
---
file: ../../../PDFREPOSITORY/000581.pdf
---
file: ../../../PDFREPOSITORY/000582.pdf
---
file: ../../../PDFREPOSITORY/000583.pdf
---
file: ../../../PDFREPOSITORY/000584.pdf
---
file: ../../../PDFREPOSITORY/000585.pdf
---
file: ../../../PDFREPOSITORY/000586.pdf
---
file: ../../../PDFREPOSITORY/000587.pdf
---
file: ../../../PDFREPOSITORY/000588.pdf
---
file: ../../../PDFREPOSITORY/000589.pdf
---
file: ../../../PDFREPOSITORY/000590.pdf
---
file: ../../../PDFREPOSITORY/000591.pdf
---
file: ../../../PDFREPOSITORY/000592.pdf
---
file: ../../../PDFREPOSITORY/000593.pdf
---
file: ../../../PDFREPOSITORY/000594.pdf
---
file: ../../../PDFREPOSITORY/000595.pdf
---
file: ../../../PDFREPOSITORY/000596.pdf
---
file: ../../../PDFREPOSITORY/000597.pdf
---
file: ../../../PDFREPOSITORY/000598.pdf
---
file: ../../../PDFREPOSITORY/000599.pdf
---
file: ../../../PDFREPOSITORY/000600.pdf
