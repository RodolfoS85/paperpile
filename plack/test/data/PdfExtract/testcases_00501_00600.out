---
authors: 'Han, Jeffrey S. and Szak, Suzanne T. and Boeke, Jef D.'
title: Transcriptional disruption by the L1 retrotransposon and implications for mammalian transcriptomes
---
authors: 'Han, Kyoungha and Yeo, Gene and An, Ping and Burge, Christopher B. and Grabowski, Paula J.'
doi: 10.1371/journal.pbio.0030158
title: 'A Combinatorial Code for Splicing Silencing: UAGG and GGGG Motifs'
---
authors: 'Hanada, Ken-ichi and Yewdell, Jonathan W. and Yang, James C.'
title: Immune recognition of a human renal cancer antigen through post-translational protein splicing
---
authors: 'Hanamura, Akemi and CÁCERES, Javier F. and Mayeda, Akila and ROBERT FRANZA, Jr., B. and Krainer, Adrian R.'
title: Regulated tissue-specific expression of antagonistic pre-mRNA splicing factors
---
authors: 'Hanash, Sam'
title: Disease proteomics
---
authors: 'Hannon, Gregory J.'
title: RNA interference
---
authors: 'Hanus, Pavol and Goebel, Bernhard and Dingel, Janis and Weindl, Johanna and Zech, Juergen and Dawy, Zaher and Hagenauer, Joachim and Mueller, Jakob C.'
doi: 10.1007/s00202-007-0062-6
title: Information and communication theory in molecular biology
---
authors: 'Hao, Bing and Gong, Weimin and Ferguson, Tsuneo K. and James, Carey M. and Krzycki, Joseph A. and Chan, Michael K.'
title: A New UAG-Encoded Residue in the Structure of a Methanogen Methyltransferase
---
authors: 'Harmanci, Arif Ozgun and Sharma, Gaurav and Mathews, David H'
doi: 10.1186/1471-2105-8-130
title: Efficient pairwise RNA structure prediction using probabilistic alignment constraints in Dynalign
---
authors: 'Hastings, Michelle L and Krainer, Adrian R'
title: Pre-mRNA splicing in the new millennium
---
authors: 'Hasty, Jeff and McMillen, David and Collins, J. J.'
title: Engineered gene circuits
---
authors: 'Hatfield, Dolph L. and Gladyshev, Vadim N.'
doi: 10.1128/MCB.22.11.3565-3576.2002
title: How Selenium Has Altered Our Understanding of the Genetic Code
---
authors: 'Hatzigeorgiou, Artemis G.'
title: Translation initiation start prediction in human cDNAs with high accuracy
---
authors: 'Havgaard, Jakob H. and Torarinsson, Elfar and Gorodkin, Jan'
doi: 10.1371/journal.pcbi.0030193
title: Fast Pairwise Structural RNA Alignments by Pruning of the Dynamical Programming Matrix
---
authors: 'Haynes, Chad and Oldfield, Christopher J. and Ji, Fei and Klitgord, Niels and Cusick, Michael E. and Radivojac, Predrag and Uversky, Vladimir N. and Vidal, Marc and Iakoucheva, Lilia M.'
doi: 10.1371/journal.pcbi.0020100
title: Intrinsic Disorder Is a Common Feature of Hub Proteins from Four Eukaryotic Interactomes
---
authors: 'Hazebrouck, Stephane and Camoin, Luc and Faltin, Zehava and Strosberg, Arthur Donny and Eshdat, Yuval'
doi: 10.1074/jbc.M004985200
title: Substituting Selenocysteine for Catalytic Cysteine 41 Enhances Enzymatic Activity of Plant Phospholipid Hydroperoxide Glutathione Peroxidase Expressed in Escherichia coli
---
authors: 'Hefferon, Timothy W. and Groman, Joshua D. and Yurk, Catherine E. and Cutting, Garry R.'
doi: 10.1073/pnas.0400182101
title: A variable dinucleotide repeat in the CFTR gene contributes to phenotype diversity by forming RNA secondary structures that alter splicing
---
authors: 'Heinemeyer, T. and Chen, X. and Karas, H. and Kel, A. E. and Kel, O. V. and Liebich, I. and Meinhardt, T. and Reuter, I. and Schacherer, F. and Wingender, E.'
title: Expanding the TRANSFAC database towards an expert system of regulatory molecular mechanisms
---
authors: 'Wingender, T. Heinemeyer E. and Hermjakob, I. Reuter H. and Kel, A. E. and Kel, O. V. and Ignatieva, E. V. and Ananko, E. A. and Podkolodnaya, O. A. and Kolpakov, F. A. and Podkolodny, N. L. and Kolchanov, N. A.'
title: 'Databases on transcriptional regulation: TRANSFAC, TRRD and COMPEL'
---
authors: 'Heinrich, Reinhart and Schuster, Stefan'
title: 'The modelling of metabolic systems. Structure, control and optimality'
---
authors: 'Heitsch, Christine E.'
title: 'The Limits of Simplicity: a Combinatorial Solution to RNA Secondary Structure Design'
---
authors: 'Hellen, Christopher U. T. and Sarnow, Peter'
doi: 10.1101/gad.891101
title: Internal ribosome entry sites in eukaryotic mRNA molecules
---
authors: 'Helling, Robert and Li, Hao and Melin, Regis and Miller, Jonathan and Wingreen, Ned and Zeng, Chen and Tang, Chao'
title: The designability of protein structures
---
authors: 'Herbert, Alan and Rich, Alexander'
title: RNA processing and the evolution of eukaryotes
---
authors: 'Crochemore, Maxime and Hermelin, Danny and Landau, Gad M. and Vialette, Stphane'
title: Approximating the 2-Interval Pattern problem
---
authors: 'Hermsen, Rutger and Tans, Sander and ten Wolde, Pieter Rein'
doi: 10.1371/journal
title: Transcriptional Regulation by Competing Transcription Factor Modules
---
authors: 'Herold, Julia and Kurtz, Stefan and Giegerich, Robert'
doi: 10.1186/1471-2105-9-167
title: Efficient computation of absent words in genomic sequences
---
authors: 'Hertel, Jana and Stadler, Peter F.'
title: 'Hairpins in a Haystack: Recognizing microRNA Precursors in Comparative Genomics Data'
---
authors: 'Hertell, Klemens J. and hlenbeck, Daniel Herschlagrand ke C. U'
title: Specificity of hammerhead ribozyme cleavage
---
authors: 'Hertz, Gerald Z. and Stormo, Gary D.'
title: Identifying DNA and protein patterns with statistically significant alignments of multiple sequences
---
authors: 'Hertzberg, Libi and Zuk, Or and Getz, Gad and Domany, Eytan'
title: Finding Motifs in Promoter Regions
---
authors: 'Ralf-Herwig,  and Poustka, Albert J. and Muller, Christine and Bull, Christof and Lehrach, Hans and OBrien, John'
title: Large-Scale Clustering of cDNA-Fingerprinting Data
---
authors: '(Pat) Heslop-Harrison, J. S.'
title: 'Planning for remodelling: nuclear architecture, chromatin and chromosomes'
---
authors: 'Heyne, Steffen and Will, Sebastian and Beckstette, Michael and Backofen, Rolf'
title: Lightweight Comparison of RNAs Based on Exact Sequence-Structure Matches
---
authors: 'Hicks, Martin J. and Yang, Chin-Rang and Kotlajich, Matthew V. and Hertel, Klemens J.'
doi: 10.1371/journal.pbio.0040147
title: Linking Splicing to Pol II Transcription Stabilizes Pre-mRNAs and Influences Splicing Patterns
---
authors: 'Hickson, Robert E. and Simon, I Chris and {“f Alan Cooper} and Spicer, Greg S. and Sullivan, Jack and Penny, David'
title: 'Conserved Sequence Motifs, Alignment, Third Domain of Animal 12s rRNA'
---
authors: 'Hickson, Robert E. and Simon, Chris and Perrey, Soren W.'
title: The Performance of Several Multiple-Sequence Alignment Programs in Relation to Secondary-Structure Features for an rRNA Sequence
---
authors: 'Hide, Winston A. and Babenko, Vladimir N. and van Heusden, Peter A. and Seoighe, Cathal and Kelso, Janet F.'
doi: 10.1101/gr.188001
title: The Contribution of Exon-Skipping Events on Chromosome 22 to Protein Coding Diversity
---
authors: 'Hill, A. A. and Hunter, C. P. and Tsung, B. T. and Tucker-Kellogg, G. and Brown, E. L.'
title: Genomic Analysis of Gene Expression in C. elegans
---
authors: 'Hiller, Michael and Zhang, Zhaiyi and Backofen, Rolf and Stamm, Stefan'
doi: 10.1371/journal.pgen
title: Pre-mRNA Secondary Structures Influence Exon Recognition
---
authors: 'Hiller, Michael and Szafranski, Karol and Sinha, Rileen and Huse, Klaus and Nikolajewa, Swetlana and Rosenstiel, Philip and Schreiber, Stefan and Backofen, Rolf and Platzer, Matthias'
doi: 10.1261/rna.883908
title: Assessing the fraction of short-distance tandem splice sites under purifying selection
---
authors: 'Hiller, Michael and Huse, Klaus and Szafranski, Karol and Jahn, Niels and Hampe, Jochen and Schreiber, Stefan and Backofen, Rolf and Platzer, Matthias'
title: Single-Nucleotide Polymorphisms in NAGNAG Acceptors Are Highly Predictive for Variations of Alternative Splicing
---
authors: 'Hiller, Michael and Huse, Klaus and Szafranski, Karol and Rosenstiel, Philip and Schreiber, Stefan and Backofen, Rolf and Platzer, Matthias'
doi: 10.1186/gb-2006-7-7-r65
title: Phylogenetically widespread alternative splicing at unusual GYNGYN donors
---
authors: 'Hiller, Michael and Szafranski, Karol and Backofen, Rolf and Platzer, Matthias'
doi: 10.1371/journal.pgen.0020045
title: 'Alternative Splicing at NAGNAG Acceptors: Simply Noise or Noise and More?'
---
authors: '{International Chicken Genome Sequencing Consortium}'
title: Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution
---
authors: 'Tyler Hillman, R and Green, Richard E and Brenner, Steven E'
title: An unappreciated role for RNA surveillance
---
authors: 'Hirao, Ichiro and Ellington, Andrew D.'
title: Re-creating the RNA world
---
authors: 'Hirosawa, Makoto and Ishikawa, Masato'
title: Comprehensive study on iterative algorithms of multiple sequence alignment
---
authors: 'Ho, Bosco K. and Dill, Ken A.'
doi: 10.1371/journal.pcbi.0020027
title: Folding Very Short Peptides Using Molecular Dynamics
---
authors: 'Hchsmann, Matthias'
title: Local Similarity in RNA Secondary Structures
---
authors: 'Hofacker, Ivo L. and Bernhart, Stephan H. F. and Stadler, Peter F.'
title: Alignment of RNA Base Pairing Probability Matrices
---
authors: 'Hofacker, Ivo L'
title: How microRNAs choose their targets
---
authors: 'Hofacker, Ivo L. and Stadler, Peter F.'
title: Automatic Detection of Conserved Base Pairing Patterns in RNA Virus Genomes
---
authors: 'Hoffmann, Steve and Otto, Christian and Kurtz, Stefan and Sharma, Cynthia M. and Khaitovich, Philipp and Vogel, Jorg and Stadler, Peter F. and Hackermuller, Jorg'
doi: 10.1371/journal.pcbi.1000502
title: 'Fast Mapping of Short Sequences with Mismatches, Insertions and Deletions Using Index Structures'
---
authors: 'Hoffmann, Jules A.'
title: The immune response of Drosophila
---
authors: 'Hogan, Daniel J. and Riordan, Daniel P. and Gerber, Andre P. and Herschlag, Daniel and Brown, Patrick O.'
doi: 10.1371/journal.pbio.0060255
title: 'Diverse RNA-Binding Proteins Interact with Functionally Related Sets of RNAs, Suggesting an Extensive Regulatory System'
---
authors: 'Höglund, Annette and Kohlbacher, Oliver'
title: 'From sequence to structure and back again: approaches for predicting protein-DNA binding'
---
authors: 'Hohm, Tim and Limbourg, Philipp and Hoffmann, Daniel'
title: A Multiobjective Evolutionary Method for the Design of Peptidic Mimotopes
---
authors: 'Holm, Liisa and Sander, Chris'
title: Touring protein fold space with Dali/FSSP
---
authors: 'Holmes, Ian'
doi: 10.1186/1471-2105-6-73
title: Accelerated probabilistic inference of RNA structure evolution
---
authors: 'Homma, Keiichi and Kikuno, Reiko F. and Nagase, Takahiro and Ohara, Osamu and Nishikawa, Ken'
doi: 10.1016/j.jmb.2004.09.028
title: Alternative Splice Variants Encoding Unstable Protein Domains Exist in the Human Brain
---
authors: 'Honig, Barry'
title: 'Protein Folding: From the Levinthal Paradox to Structure Prediction'
---
authors: 'Hooper, Peter M. and Zhang, Haiyan and Wishart, David S.'
title: Prediction of genetic structure in eukaryotic DNA using reference point logistic regression and sequence alignment
---
authors: 'Hooper, Nigel M. and Karran, Eric H. and Turner, Anthony J.'
title: Membrane protein secretases
---
authors: 'Hoque, Tamjidul and Chetty, Madhu and Sattar, Abdul'
doi: 10.1089/cmb.2008.0082
title: Extended HP Model for Protein Structure Prediction
---
authors: 'Horesh, Yair'
title: A bottom-up clustering algorithm to detect ncRNA molecules with a common secondary structure
---
authors: 'Horiuchi, Takayuki and Aigaki, Toshiro'
doi: 10.1042/BC20050002
title: 'Alternative trans-splicing: a novel mode of pre-mRNA processing'
---
authors: 'Hoshiyama, Daisuke and Kuma, Kei-ichi'
title: Markedly Reduced Evolutionary Rates of Transcription Factors and Cytoplasmic Ribosomal RNAs and Proteins in Higher Vertebrates and Their Evolutionary Implications
---
authors: 'Lynch, Amy E. Houseand Kristen W.'
doi: 10.1074/jbc.R700031200
title: 'Regulation of Alternative Splicing: More than Just the ABCs'
---
authors: 'Hsu, Hsiao-Ping and Mehra, Vishal and Nadler, Walter and Grassberger, Peter'
doi: 10.1103/PhysRevE.68.021113
title: Growth-based optimization algorithm for lattice heteropolymers
---
authors: 'Hu, Yuh-Jyh'
title: Prediction of consensus structural motifs in a family of coregulated RNA sequences
---
authors: 'Huang, Xiaoqiu and Chao, Kun-Mao'
title: A generalized global alignment algorithm
---
authors: 'Huang, Haiming and Li, Lei and Wu, Chenggang and Schibli, David and Colwill, Karen and Ma, Sucan and Li, Chengjun and Roy, Protiva and Ho, Krystina and Songyang, Zhou and Pawson, Tony and Gao, Youhe and -C. Li, Shawn S.'
title: Defining the Specificity Space of the Human Src-homology 2 Domain
---
authors: 'Huang, Weichun and Marth, Gabor'
doi: 10.1101/gr.076067.108
title: 'EagleView: A genome assembly viewer for next-generation sequencing technologies'
---
authors: 'Huang, Enoch S. and Samudrala, Ram and Ponder, Jay W.'
title: Ab Initio Fold Prediction of Small Helical Proteins using Distance Geometry and Knowledge-based Scoring Functions
---
authors: 'Huang, FRANÇOISE'
title: Syntagms in development and evolution
---
authors: 'Huang, Hsien-Da and Horng, Jorng-Tzong and Lee, Chau-Chin and Liu, Baw-Jhiune'
title: 'ProSplicer: a database of putative alternative splicing information derived from protein, mRNA and expressed sequence tag sequence data'
---
authors: 'Hubbard, T. and Barker, D. and Birney, E. and Cameron, G. and Chen, Y. and Clark, L. and Cox, T. and Cuff, J. and Curwen, V. and Down, T. and Durbin, R. and Eyras, E. and Gilbert, J. and Hammond, M. and Huminiecki, L. and Kasprzyk, A. and Lehvaslaiho, H. and Lijnzaad, P. and Melsopp, C. and Mongin, E. and Pettett, R. and Pocock, M. and Potter, S. and Rust, A. and Schmidt, E. and Searle, S. and Slater, G. and Smith, J. and Spooner, W. and Stabenau, A. and Stalker, J. and Stupka, E. and Ureta-Vidal, A. and Vastrik, I. and Clamp, M.'
title: The Ensembl genome database project
---
authors: 'Huez, Isabelle and Creancier, Laurent and Audigier, Sylvie and Gensac, MARIE-CLAIRE and Prats, ANNE-CATHERINE and Prats, Herve'
title: Two Independent Internal Ribosome Entry Sites Are Involved in Translation Initiation of Vascular Endothelial Growth Factor mRNA
---
authors: 'Hung, Li-Wei and Holbrook, Elizabeth L. and Holbrook, Stephen R.'
doi: 10.1073/pnas.090588197
title: The crystal structure of the Rev binding element of HIV-1 reveals novel base pairing and conformational variability
---
authors: 'Kibler, Dennis and Hampson, Steven'
title: 'Characterizing the Shine-Dalgarno Motif: Probability Matrices and Weight Matrices'
---
authors: 'Huttenhofer, Alexander and Bock, August'
title: Selenocysteine Inserting RNA Elements Modulate GTP Hydrolysis of Elongation Factor SelB
---
authors: 'Hüttenhofer, Alexander and Brosius, Jürgen and Bachellerie, Jean-Pierre'
doi: 10.1016/S1367-59310200397-6
title: 'RNomics: identification and function of small, non-messenger RNAs'
---
authors: 'Hüttenhofer, Alexander and Heider, Johann and Böck, August'
title: Interaction of the Escherichia coli fdhF mRNA hairpin promoting selenocysteine incorporation with the ribosome
---
authors: 'Hutvágner, György and Zamore, Phillip D'
title: 'RNAi: nature abhors a double-strand'
---
authors: 'H, Martijn A. and Stadler, Peter F. and Fontana, Walter'
title: 'Smoothness within ruggedness: The role of neutrality in adaptation'
---
authors: 'Zhang, Harry and Su, Jiang'
title: Conditional Independence Trees
---
authors: '{International Human Genome Sequencing Consortium}'
title: Initial sequencing and analysis of the human genome
---
authors: 'Iorio, Marilena V. and Visone, Rosa and Di Leva, Gianpiero and Donati, Valentina and Petrocca, Fabio and Casalini, Patrizia and Taccioli, Cristian and Volinia, Stefano and Liu, Chang-Gong and Alder, Hansjuerg and Calin, George A. and Menard, Sylvie and Croce, Carlo M.'
doi: 10.1158/0008-5472.CAN-07-1936
title: MicroRNA Signatures in Human Ovarian Cancer
---
authors: 'Ioshikhes, Ilya and Trifonov, Edward N. and Zhang, Michael Q.'
title: Periodical distribution of transcription factor sites in promoter regions and connection with chromatin structure
---
arxivid: cond-mat/0201009
authors: 'Irbck, Anders and Troein, Carl'
title: Enumerating Designing Sequences in the HP Model
---
authors: 'Isalan, Mark and Klug, Aaron and Choo, Yen'
title: Comprehensive DNA Recognition through Concerted Interactions from Adjacent Zinc Fingers
---
authors: 'Istrail, Sorin and Davidson, Eric H.'
doi: 10.1073/pnas.0409624102
title: Logic functions of the genomic cis-regulatory code
---
authors: 'Itoh, Hitomi and Washio, Takanori and Tomita, Masaru'
doi: 10.1261/rna.5221604
title: Computational comparative analyses of alternative splicing regulation using full-length cDNA of various eukaryotes
---
authors: 'Jabbari, Hosna and Condon, Anne and Pop, Ana and Pop, Cristina and Zhao, Yinglei'
title: 'HFold: RNA Pseudoknotted Secondary Structure Prediction Using Hierarchical Folding'
---
authors: 'Jabri, Evelyn and Aigner, Stefan and Cech, Thomas R.'
title: 'Kinetic and Secondary Structure Analysis of Naegleria andersoni GIR1, a Group I Ribozyme Whose Putative Biological Function Is Site-Specific Hydrolysis'
---
authors: 'Zongker, Douglas'
title: 'Algorithms for Feature Selection: An Evaluation'
---
authors: 'Jain, Anil and Zongker, Douglas'
title: 'Feature Selection: Evaluation, Application, and Small Sample Performance'
---
authors: 'Jankowsky, Eckhard and Gross, Christian H. and Shuman, Stewart and Pyle, Anna Marie'
title: Active Disruption of an RNA-Protein Interaction by a DExH/D RNA Helicase
---
authors: 'Jans, David A. and Xiao, Chong-Yun and Lam, Mark H. C.'
title: 'Nuclear targeting signal recognition: a key control point in nuclear transport?'
